### ISSUE TITLE
[UI] add UI & API version to footer

### ISSUE THREAD

**Issue by samrichca** [icgc-argo/dac-ui#136]
add UI & API version to footer

**Comment by kcullion** [comment]
@samrichca will this eventually be fixed? the vundefined...
Also is "bb007ad" part of it? I see it's different on the platform so just curious
![image](https://user-images.githubusercontent.com/18424138/123297017-28ada500-d4e5-11eb-803a-dd02dad4a95e.png)

**Comment by samrichca** [comment]
> @samrichca will this eventually be fixed? the vundefined...
> Also is "bb007ad" part of it? I see it's different on the platform so just curious
> ![image](https://user-images.githubusercontent.com/18424138/123297017-28ada500-d4e5-11eb-803a-dd02dad4a95e.png)

@blabadi ❓🙋‍♀️

**Comment by blabadi** [comment]
@kcullion yeah I will fix the api version you can assign this ticket to me

**Comment by kcullion** [comment]
thanks @blabadi I added you as assignee and moved to in progress

**Comment by blabadi** [comment]
I'm creating a new ticket for the backend to unblock this ticket from testing

**Comment by kcullion** [comment]
@blabadi this looks good to me! thanks

**Comment by rosibaj** [comment]
verified with backend: 
![image](https://user-images.githubusercontent.com/15824252/123657994-4f2e5180-d7ff-11eb-97fa-f6a4e3db03be.png)
